epics/pv.py

```diff
@@ -356,10 +356,22 @@ def _set_charval(self, val, call_ca=True):
         # char waveform as string
         if ntype == dbr.CHAR and self.count < ca.AUTOMONITOR_MAXLENGTH:
             if isinstance(val, ca.numpy.ndarray):
```

Comment:
I attempted to fix this in the other PR - but am purposefully leaving it out in this one to limit the scope. 

This will fail miserably without numpy installed as there is no check for it. I think it would be a good idea to either consider making numpy a requirement for pyepics, or always run tests in two separate environments - with and without numpy.